Makefile

```diff
@@ -388,64 +441,120 @@ st: dist/calicoctl busybox.tar routereflector.tar calico-node.tar workload.tar r
 	           -e HOST_CHECKOUT_DIR=$(CURDIR) \
 	           -e DEBUG_FAILURES=$(DEBUG_FAILURES) \
 	           -e MY_IP=$(LOCAL_IP_ENV) \
-	           -e NODE_CONTAINER_NAME=$(NODE_CONTAINER_NAME) \
-             -e RR_CONTAINER_NAME=$(RR_CONTAINER_NAME):$(RR_VER) \
+		   -e NODE_CONTAINER_NAME=$(NODE_CONTAINER_NAME):latest-$(ARCH) \
```

Comment:
Change to the indent here (2 tabs) makes it inconsistent with the line above (1 tab then spaces). The latter is my preferred style since it lets you align arguments visually.